rialto_airflow/harvest/openalex.py

```diff
@@ -100,7 +100,8 @@ def publications_from_dois(dois: list):
                     pubs = Works().filter(doi=doi).get()
                     if len(pubs) > 1:
                         logging.warn(f"Found multiple publications for DOI {doi}")
-                    yield normalize_publication(pubs[0])
+                    if pubs:
```

Comment:
Should this be an `else`?

Reply:
Actually I understand now. Maybe it should be `if len(pubs) > 0` ?

Reply:
Made that change